### ISSUE TITLE
Intel Cards wont mine anymore

### ISSUE THREAD

**Issue by Darklegion1978** [Lolliedieb/lolMiner-releases#2119]
Since the new Firmware Update on Intel Cards A770/A750, mining wont work anymore.
Programm starts, and short after the first run, Windows freeze.

With the old Firmware on the Cards it works like a charme, but not with the new Bios Update.

New Firmware = [DG02_1.3266]
Old Firmware = [DG02_1.3257]

For all they have troubles mining with Intel Arc, use the old Firware!

Greetings

**Comment by jgonzis** [comment]
> Since the new Firmware Update on Intel Cards A770/A750, mining wont work anymore. Programm starts, and short after the first run, Windows freeze.
> 
> With the old Firmware on the Cards it works like a charme, but not with the new Firmware.
> 
> New Firmware = [DG02_1.3266] Old Firmware = [DG02_1.3257]
> 
> For all they have troubles mining with Intel Arc, use the old Firware!
> 
> Greetings

It directly freezes? or says something? Any option to run clinfo to have some extra information?

**Comment by Darklegion1978** [comment]
Yes, after the start of Lolminer, it checks the available cards.
Found the Intel Card and start with mining, but directly after the start of mining, the complete system freeze.

At the moment im not at home, but when i have the chance to try clinfo, i give it a try.

BTW SRBMiner works as like before, without any issues.

Greets

**Comment by Darklegion1978** [comment]
I tried a little after im back home.

The failure is just with the algo Beamhash.
It starts mining, but short after it crashes.

With algo etc, lolminer works well.

Greetings
